securenative.gemspec

```diff
@@ -6,7 +6,7 @@ require_relative 'lib/securenative/utils/version_utils'
 
 Gem::Specification.new do |spec|
   spec.name          = 'securenative'
-  spec.version       = '0.1.28'
+  spec.version       = '0.1.29'
   spec.authors       = ['SecureNative']
   spec.email         = ['support@securenative.com']
 
```

Comment:
I've added `spec. test_files`, spec folder also contains a spec help file which defines tests.

This file was autogenerated by IntelliJ, I guess it's they're way of specifying files (using git), anyway i don't know if using `spec.files = Dir['{lib}//*'] + ['README.md']` will do it, still missing important files like `Gemfile` and `securenative.spec` not sure it will work bundles like that